project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
I needed `org.clojure/java.jmx` to be included during test runs b/c without it, I'd get this failure when I tried to run `lein test`:

~~~~
Exception in thread "main" java.io.FileNotFoundException: Could not locate clojure/java/jmx__init.class or clojure/java/jmx.clj on classpath., compiling:(puppetlabs/trapperkeeper/testutils/webserver.clj:1:1)
...
	at puppetlabs.puppetserver.ruby.http_client_test$eval24317$loading__5340__auto____24318.invoke(http_client_test.clj:1)
	at puppetlabs.puppetserver.ruby.http_client_test$eval24317.invoke(http_client_test.clj:1)
...
~~~~

Looks like jmx is only used in tk-jetty9 as a dev dependency for tests as well.  So apparently we don't get it transitively from either tk-jetty9 or tk-jetty9's testutils jars?

Should I have resolved this in a different way?